<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reactiichimice</class>
 <widget class="QMainWindow" name="reactiichimice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reactii Chimice</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>331</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Calculator Solutii</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>751</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Calculator Reactii</string>
    </property>
    <widget class="QPlainTextEdit" name="balanceoutput">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="balanceinput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Introduceti 2 reactanti</string>
     </property>
    </widget>
    <widget class="QPushButton" name="balancebtn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculeaza</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>751</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Balanseaza</string>
    </property>
    <widget class="QLabel" name="elem1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cof1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>42</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="pluslabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="elem2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cof2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>42</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="arrowlabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="elem3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cof3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>42</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="balt">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Altceva</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bverifica">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Verifica</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
